### PR TITLE
foo

### PR DESCRIPTION
**Pull request by ngnmnfn**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/ngnmnfn/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: test [hunk 1]
```diff
@@ -0,0 +1,11 @@
+When has justice ever been as simple as a rule book? Flair is what marks the difference between artistry and mere competence.
```
Comment (ngnmnfn) [inline test]: line note!

File: test [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ You bet I'm agitated! I may be surrounded by insanity, but I am not insane.
+ Our neural pathways have become accustomed to your sensory input patterns.
+ I'm afraid I still don't understand, sir.
+ We finished our first sensor sweep of the neutral zone.
+ Ensign Babyface! Now we know what they mean by 'advanced' tactical training.
```
Comment (ngnmnfn) [inline test]: !fix this shouldn't be here

File: test [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ And blowing into maximum warp speed, you appeared for an instant to be in two places at once.
+ Well, that's certainly good to know.
+ Is it my imagination, or have tempers become a little frayed on the ship lately? I'd like to think that I haven't changed those things, sir.
+ Maybe we better talk out here; the observation lounge has turned into a swamp.
```
Comment (ngnmnfn) [inline test]: !fix this is a thing to do, too

File: test (hunk 2 above)
Comment (ngnmnfn) [inline test]: !fixed removed it

File: test (hunk 1 above)
Comment (ngnmnfn) [inline test]: !resolved

File: test (hunk 3 above)
Comment (ngnmnfn) [inline test]: !fixed

File: test [hunk 4]
```diff
@@ -0,0 +1,11 @@
+When has justice ever been as simple as a rule book? Flair is what marks the difference between artistry and mere competence.
+ You bet I'm agitated! I may be surrounded by insanity, but I am not insane.
+ Our neural pathways have become accustomed to your sensory input patterns.
+ I'm afraid I still don't understand, sir.
+ We finished our first sensor sweep of the neutral zone.
+ Ensign Babyface! Now we know what they mean by 'advanced' tactical training.
+ And blowing into maximum warp speed, you appeared for an instant to be in two places at once.
```
Comment (ngnmnfn) [inline test]: htnshtn

Comment (ngnmnfn) [inline test]: !fix

Comment (ngnmnfn) [inline test]: !fixed

**Pull request merged**